### DISCUSSION TITLE
color_thresholds values with greater or less than

### DISCUSSION THREAD

**Discussion by Pardasus** [kalkih/mini-graph-card#1178] [Ideas]
Hi,
I would like to be able to enter values with from/to in the values for colour_thresholds.

For example, power consumption:
0-100 watts = green
100-500 watts = yellow
500-1,000 watts = red

**Comment by ildar170975** [comment]
Here is a good place to ask questions for "how to use the card?"
https://community.home-assistant.io/t/lovelace-mini-graph-card/71385
